2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -0,0 +1,203 @@
+  ),
+  showIndex: true,
+);
+#set text(font:"New Computer Modern")
+#set align(left)
+#show "link": word => text[#text(fill:blue, word)]
+
+= Introduzione
+Il documento _Piano di progetto_ ha il delicato compito di governare la pianificazione dell'avanzamento del progetto, determinando task e obiettivi da raggiungere, presentando un'analisi critica del lavoro fino a quel momento svolto. L'intento è rendicontare l'intero sviluppo, valutando criticamente l'operato compiuto per eventualmente miglirarlo, e gestire in modo efficace ed efficiente le risorse.
+
+Il documento si articola in 5 sezioni, quali:
+- *Rischi e loro mitigazione*: dedicata all'analisi dei rischi che possono insorgere con relativi sistemi di mitigazioni;
+- *Divisione temporale di sviluppo*: dedicata all'analisi della suddivisione temporale dello sviluppo del progetto;
+- *Preventivo dei costi di realizzazione*: dedicata all'analisi preventiva dei costi di realizzazione di progetto;
+- *Pianificazione del lavoro*: dedicata alla pianificazione delle attività da svolgere nei periodi di sprint;
+- *Consuntivo del progetto*: dedicata all'analisi retrospettiva del lavoro svolto, indicando criticità ed eventuali azioni intraprese a fini migliorativi.
+\
+== Glossario
+Al fine di rendere quanto più chiaro possibile il contenuto dei documenti, e per evitare qualsiasi forma di ambiguità in merito ai termini utilizzati, il gruppo _Error\_418_ fornirà come risorse web consultabile, un _*Glossario*_. La prima ricorrenza di termini tecnici appartenenti al dominio del progetto, saranno caratterizzati dalla presenza della lettera _g_ al pedice.
+\
+\
+== Riferimenti esterni
+=== Norme
+- Regolamento di progetto \
+  _#link("https://www.math.unipd.it/~tullio/IS-1/2023/Dispense/PD2.pdf")_
+
+- Gestione di progetto \
+  _#link("https://www.math.unipd.it/~tullio/IS-1/2023/Dispense/T4.pdf")_
+
+- I processi di ciclo di vita del software \
+  _#link("https://www.math.unipd.it/~tullio/IS-1/2023/Dispense/T2.pdf")_
+
+=== Capitolato
+- Capitolato C5: WMS3: Warehouse management 3D \
+  _#link("https://www.math.unipd.it/~tullio/IS-1/2023/Progetto/C5.pdf")_
+
+#pagebreak()
+
+= Rischi e loro mitigazione
+= Divisione temporale di sviluppo
+== Introduzione
+== Periodi di sviluppo
+=== Periodo RTB
+- Analisi dei requisiti
+- Proof of Concept
+- date nostra suddivisione
+=== Periodo PB
+Ancora da definire
+=== Periodo CA
+Ancora da definire
+= Preventivo dei costi di realizzazione
+== Introduzione
+Definiti i rischi (e le relative contromisure) e il calendario di progetto, si è definito, mediante una pianificazione retrospettiva, un preventivo iniziale dei costi di realizzazione del progetto. \
+È altresì corretto evidenziare come i membri del gruppo non siano dotati di esperienza sufficiente per fornire un preventivo corretto e preciso sin dagli inizi dello sviluppo: per tale motivo, il prezzo indicato sarà soggetto a modifiche con l'avanzamento del progetto.
```

Comment:
Qui bisogna stare attenti, perché 1) da regolamento il prezzo non può aumentare 2) non so se il prezzo possa fluttuare anche dopo l'implementazione del feedback ricevuto sulla Candidatura.
```suggestion
È altresì corretto evidenziare come i membri del gruppo non siano dotati di esperienza sufficiente per fornire un preventivo corretto e preciso sin dagli inizi dello sviluppo: per tale motivo, il prezzo indicato sarà soggetto a modifiche con l'avanzamento del progetto.
```

Reply:
1) specificato che non può superare il prezzo preventivato in candidatura
2) Sì, chiesto al professore